handsontable/src/pluginHooks.js

```diff
@@ -74,15 +121,29 @@ const REGISTERED_HOOKS = [
    * @param {Array} changes 2D array containing information about each of the edited cells `[[row, prop, oldVal, newVal], ...]`.
    * @param {string} [source] String that identifies source of hook call ([list of all available sources](@/guides/getting-started/events-and-hooks.md#definition-for-source-argument)).
    * @example
+   * ::: only-for javascript
    * ```js
    * new Handsontable(element, {
    *   afterChange: (changes) => {
-   *     changes.forEach(([row, prop, oldValue, newValue]) => {
+   *     changes?.forEach(([row, prop, oldValue, newValue]) => {
    *       // Some logic...
    *     });
    *   }
    * })
    * ```
+   * :::
+   *
+   * ::: only-for react
+   * ```jsx
+   * <HotTable
+   *   afterChange={(changes, source) => {
+   *     changes.forEach(([row, prop, oldValue, newValue]) => {
```

Comment:
You added `changes?.` for the vanilla version, but not the react one

Reply:
Sorry for not describing it. I assumed, it is very likely TypeScript or any module bundler will be enabled for React while it it not so certain that this code be handled by vanilla. Without optional chaining operator `(?.)`, the TS shows an error.

However, it seems that [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) works for every modern version of browser meeting our [supported browsers criteria.](https://handsontable.com/docs/supported-browsers/#testing-compatibility) I'll add it also for JS snippets.

Reply:
Changed within: https://github.com/handsontable/handsontable/commit/12fd2247cd60b09fd16030f010286a80376f4cf5#diff-9c8662f02ad3bdda7b952d5aba2a3e3e1ee8ca2600f3bedffb0f3781022992a1R148